operator/go.mod

```diff
@@ -3,7 +3,11 @@ module k8s.io/kubeadm/operator
 go 1.12
 
 require (
+	github.com/go-logr/logr v0.1.0
+	github.com/onsi/ginkgo v1.6.0
+	github.com/onsi/gomega v1.4.2
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	k8s.io/utils v0.0.0-20190506122338-8fab8cb257d5
```

Comment:
with kubeadm, kinder and operator we are *probably* going to end up in a weird situation where each projects needs a different version of libs like api-machinery / client-go. ;)

Reply:
weel they are three separate projects that we are hosting in the same repo, so I don't see this so weird.

the alternative, have a unique go.mod file for kubeadm, kubeadm-operator and kinder is even less attractive to me because it mixes up things which are very different, but we should consider simplifying hacks, makefiles, godep, builds at a later stage when also kubeadm lands in this repo

Reply:
they are separate projects, true.
one potential convergence is the operator moving into kubeadm.